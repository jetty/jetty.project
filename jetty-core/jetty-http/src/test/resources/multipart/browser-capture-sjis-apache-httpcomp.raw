--u7tfLQaHJEHHUJjnVDbFdc_Oqz4jmkA25mgWd
Content-Disposition: form-data; name="japanese"
Content-Type: text/plain; charset=Shift_JIS
Content-Transfer-Encoding: 8bit

オープンソース
--u7tfLQaHJEHHUJjnVDbFdc_Oqz4jmkA25mgWd
Content-Disposition: form-data; name="hello"
Content-Type: text/plain; charset=Shift_JIS
Content-Transfer-Encoding: 8bit

日食桟橋
--u7tfLQaHJEHHUJjnVDbFdc_Oqz4jmkA25mgWd--
