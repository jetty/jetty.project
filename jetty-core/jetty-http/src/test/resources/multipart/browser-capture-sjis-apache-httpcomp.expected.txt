Request-Header|Accept-Encoding|gzip,deflate
Request-Header|Connection|keep-alive
Request-Header|Content-Length|406
Request-Header|Content-Type|multipart/form-data; boundary=u7tfLQaHJEHHUJjnVDbFdc_Oqz4jmkA25mgWd
Request-Header|Host|localhost:9090
Request-Header|User-Agent|Apache-HttpClient/4.5.5 (Java/1.8.0_162)
Request-Header|X-BrowserId|apache-httpcomp
Parts-Count|2
Part-ContainsContents|japanese|オープンソース
Part-ContainsContents|hello|日食桟橋